src/routes/Apps/App/Settings.tsx

```diff
@@ -211,8 +211,7 @@ class Settings extends React.Component<Props, ComponentState> {
             // packageVersions: DON'T SEND
             // devPackageId: DON'T SEND
             trainingFailureMessage: null,
-            trainingStatus: TrainingStatusCode.Completed,
-            datetime: new Date()
+            trainingStatus: TrainingStatusCode.Completed
```

Comment:
I had originally though this `datetime` would only be for the last training update, but this was also updated for settings.

Did we want to keep that so when you change settings it updates this time?

Reply:
Brings up my related question from Teams, it seems we added `lastModifiedDateTime` after `datetime` was here and now they seem to be updated inconsistently and it's not clear which is for what.

Since `datetime` is updated from Settings page update it is kind of a `lastModifiedDateTime`

The extra date time seems to make this confusing.